xml/System.Runtime.CompilerServices/HasCopySemanticsAttribute.xml

```diff
@@ -42,7 +42,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>This class is not used in the .NET Framework version 2.0 and is reserved for future use. This class cannot be inherited.</summary>
+    <summary>This class is reserved for future use. This class cannot be inherited.</summary>
```

Comment:
I wonder if this summary is obsolete now, and we are already using it.

Reply:
@tgani-msft Is this description for `HasCopySemanticsAttribute` still accurate?

Reply:
@gewarren this attribute is obsolete and neither referenced nor emitted by the C++/CLI compiler.